herring/puzzles/models.py

```diff
@@ -17,9 +38,11 @@ def __str__(self):
 
     class Meta:
         ordering = ['number']
+    class Json:
+        include_fields = ['name', 'number']
```

Comment:
It seems a bit overengineered/redundant to include a class to specify which fields to include given that in all cases we dump every model field.

Could JSONMixin serialize all fields by default, unless there happens to be a Json class present?